e2e/a11y/pages/FormField.tsx

```diff
@@ -0,0 +1,48 @@
+import { FormField } from '@spark-ui/form-field'
+import React from 'react'
+
+export const A11yFormField = () => (
+  <section>
+    <div>
+      <FormField name="email">
+        <FormField.Label>Email</FormField.Label>
+
+        <FormField.Control>
+          {({ id, name, description }) => (
+            <input
```

Comment:
you should use a `@spark-ui/input` here

Reply:
Why? This test should focus on form-field, shouldn't it?

Reply:
Yes, but why not make sure the `Formfield` works well with our input at the same time?

Reply:
Tbh I'm not sure... In this case shouldn't we add all our input components within Formfield implementation? :thinking: